src/utilities/flex/TransientTokenGenerator.js

```diff
@@ -0,0 +1,19 @@
+
+'use strict';
+
+function TransientTokenGenerator(jwt)
+{
+    var splitContents = jwt.split(".");
+    if(splitContents.length > 1)
+    {
+        var encodedString  = splitContents[1];
+        var data = new Buffer(encodedString);
+        var decodedString =data.toString('base64');
+        var transientTokenModel= JSON.parse(decodedString);
+        // return JTI string
+        return transientTokenModel;
+    }
+
+}
+
+module.exports = TransientTokenGenerator;
```

Comment:
The name of a function should always be an action, like `verifyToken()` or `parseToken()`.

Change the structure of this file to mimic the same structure of `TokenVerification.js`